cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -356,7 +356,7 @@ func (c Client) WaitForClusterAvailable(delay time.Duration, retries int, retryI
 	return false, errors.New("timeout waiting for etcd cluster to be available")
 }
 
-// CheckConfigurationIsHA returns true if the given InitConfiguration etcd block appears to be an HA configuration.
+// CheckConfigurationIsHA returns true if the given init configuration etcd block appears to be an HA configuration.
```

Comment:
InitConfiguration is a struct, better to keep as is

Reply:
+1

Reply:
same to `EtcdCluster`